src/System.Net.Security/src/System.Net.Security.csproj

```diff
@@ -99,6 +101,24 @@
       <Link>Common\System\Net\SecurityStatusPal.cs</Link>
     </Compile>
   </ItemGroup>
+  <!-- This file depends on IANA registry. We do not want anyone's build to break after the update -->
+  <!-- or if they don't have internet connection - explicit opt-in required -->
+  <!-- To expose newly generated APIs, generated file have to be deliberately copied -->
+  <ItemGroup Condition="'$(AllowTlsCipherSuiteGeneration)' == 'true'">
+    <Compile Include="System\Net\Security\TlsCipherSuite.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>TlsCipherSuite.tt</DependentUpon>
+    </Compile>
+    <None Include="System\Net\Security\TlsCipherSuite.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>TlsCipherSuite.cs</LastGenOutput>
+    </None>
+  </ItemGroup>
+  <ItemGroup Condition="'$(AllowTlsCipherSuiteGeneration)' != 'true'">
+    <Compile Include="System\Net\Security\TlsCipherSuite.cs" />
```

Comment:
Should this get a DependentUpon entry, like it does when AllowTlsCipherSuiteGeneration is true?

Reply:
We don't want any generation to happen in that case right? (unless this only nests it under VS and nothing else then can change)

Reply:
Yeah, the DependentUpon in this case would just show it as a nested file in the solution explorer.

Reply:
ok, I was able to simplify these a bit (although not everything) - added DependsUpon regardless of AllowTlsCipherSuiteGeneration